releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -6,6 +6,14 @@
 
 [Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
 
+## 1.1.5/1.2.5 - April 27, 2020
+
+* Added support to allow users to configure the client used to connect to App Configuration and Key Vault. Users can now define the clients via `ConfigurationClientBuilderSetup` and `SecretClientBuilderSetup`, shown [here](https://github.com/microsoft/spring-cloud-azure/tree/master/spring-cloud-azure-starters/spring-cloud-starter-azure-appconfiguration-config#modifying-connection-client).
+  * This allows for setting up additional client information such as proxy information. [#656](https://github.com/microsoft/spring-cloud-azure/issues/656)
+* Fixed the bug that caused a partial configuration to load when both `/application/` and `/<application_name>/` are used, and only one of them loaded. This also caused the store to be unable to refresh.
```

Comment:
>* Fixed the bug that caused a partial configuration to load when both `/application/` and `/<application_name>/` are used, and only one of them loaded. This also caused the store to be unable to refresh [](start = 0, length = 202)

"This also caused" in the second sentence is confusing since the first sentence talks about "Fixed the bug". How about the following?

Fixed the bug that caused configuration to load partially and refresh to fail when both `/application/` and `/<application_name>/` were used but only one of them loaded.